design-system-docs/src/_layouts/default.erb

```diff
@@ -24,6 +23,7 @@
         { url: "/components", title: "Components" }
       ]
     ) %>
+    <%= render Shared::AppBreadcrumbs.new(resource) %>
```

Comment:
Might be worth putting the alpha notice banner here too, so it's always everywhere?